website/docs/developer/research/research-orca-delete-functionality.md

```diff
@@ -0,0 +1,122 @@
+---
```

Comment:
This shouldn't be a part of this PR, It should be added to a different PR.

Reply:
sorry got uploaded by mistake